tests/memory_tests/src/memory_tests/memtest_infer.cpp

```diff
@@ -15,8 +15,12 @@
  * main(). The function should not throw any exceptions and responsible for
  * handling it by itself.
  */
-int runPipeline(const std::string &model, const std::string &device) {
-    auto pipeline = [](const std::string &model, const std::string &device) {
+int runPipeline(const std::string &model, const std::string &device,
+                std::map<std::string, ov::PartialShape> reshapeShapes,
+                std::map<std::string, std::vector<size_t>> dataShapes) {
+    auto pipeline = [](const std::string &model, const std::string &device,
+                       std::map<std::string, ov::PartialShape> reshapeShapes,
+                       std::map<std::string, std::vector<size_t>> dataShapes) {
```

Comment:
just a minor note: `ov::Shape` can be used here instead of `std::vector<size_t>`